### DISCUSSION TITLE
Question about FCST_VARn, OBS_VARn, and/or BOTH_VARn

### DISCUSSION THREAD

**Discussion by michelleharrold** [dtcenter/METplus#1270] [Configuration] [answered]
Good Morning,

I am modifying some pre-existing METplus configuration files. In some circumstances, the name, levels, and/or thresh fields are set to BOTH_VAR*, as opposed to FCST_VAR* and OBS_VAR* -- but sometimes there are additional options that are set, e.g., BOTH_VAR1_OPTIONS = interp = { type = [ { method = NEAREST; width = 1; } ]; } that only occur in the FCST or OBS field (in this case OBS) but are still set as BOTH in the configuration file. MET runs fine and produces the desired output in these cases (so far). From a best practices perspective, is it better to separate out the BOTH_ into FCST_ and OBS_? Also can you have a combination of BOTH_VARn_, FCST_VARn, and OBS_VARn?

Thanks!
Michelle

**Comment by michelleharrold** [comment]
Just as a follow-up, I tested a combination of BOTH_VAR* with OBS_VAR*, and MET errored:

`ERROR: Cannot set FCST_VAR3_OPTIONS or OBS_VAR3_OPTIONS if BOTH_VAR3_OPTIONS is set.`

I also did some testing and landed on separating out BOTH_VAR* into FCST_VAR* and OBS_VAR*.

I think I am set with my configuration files, thanks!
Michelle

**Comment by georgemccabe** [comment, marked as answer]
Hi @michelleharrold, In general I would only use BOTH_ variables for a given VAR<n> if all of the settings are identical for both FCST and OBS. If the forecast and observation information differ in any way, I would recommend splitting them up into FCST_ and OBS_ variables.

**Reply by michelleharrold** [reply]
@georgemccabe - thanks for confirming it is best to split them into FCST_ and OBS_!
